node/executor/src/lib.rs

```diff
@@ -318,9 +319,9 @@ mod tests {
 			sudo: Some(Default::default()),
 			grandpa: Some(GrandpaConfig {
 				authorities: vec![ // set these so no GRANDPA events fire when session changes
-					(keyring::ed25519::Keyring::Charlie.to_raw_public().into(), 1),
-					(keyring::ed25519::Keyring::Bob.to_raw_public().into(), 1),
-					(keyring::ed25519::Keyring::Alice.to_raw_public().into(), 1),
+					// (keyring::ed25519::Keyring::Charlie.to_raw_public().into(), 1),
```

Comment:
Commented code

Reply:
This is left to be checked by @rphmeier or you. It certainly does not break anything currently but I was a bit uncertain if this was a right action or not.

Reply:
resolved.